storybook/src/components/TextArea/TextArea.stories.tsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { TextArea, TextAreaProps } from '@amsterdam/design-system-react'
+import { useArgs } from '@storybook/preview-api'
+import { Meta, StoryObj } from '@storybook/react'
+import { exampleParagraph } from '../shared/exampleContent'
+
+const paragraph = exampleParagraph()
+
+type StoryProps = TextAreaProps & { invalid?: boolean }
+
+const meta = {
+  title: 'Components/Forms/Text Area',
+  component: TextArea,
+  args: {
+    children: paragraph,
```

Comment:
[textarea doesn't accept children in React](https://react.dev/reference/react-dom/components/textarea#props), you should set the contents with `value` or `defaultValue`